encrypted-media-respec.html

```diff
@@ -1863,7 +1867,7 @@
                   </p>
                 </li>
                 <li><p>If the preceding step failed, or if <var>sanitized session ID</var> is empty, reject <var>promise</var> with a newly created <a def-id="TypeError"></a>.</p></li>
-                <li><p>If there is an unclosed session in this object's <a def-id="document-concept"></a> whose <a def-id="sessionId"></a> attribute is <var>sanitized session ID</var>, reject <var>promise</var> with a <a def-id="QuotaExceededError"></a>.</p>
+                <li><p>If there is a <a>MediaKeySession</a> object in this object's <a def-id="document-concept"></a> whose <a def-id="sessionId"></a> attribute is <var>sanitized session ID</var>, and that is not <a def-id="media-key-session-closed"></a>, reject <var>promise</var> with a <a def-id="QuotaExceededError"></a>.</p>
```

Comment:
It might read better to say "that is not closed" first. I didn't suggest this initially since the same session ID is the most relevant part, but the order is swapped this way at line 1883. I suggest using the same order in both places.